x/auth/module.go

```diff
@@ -218,6 +220,8 @@ type ModuleInputs struct {
 
 	// LegacySubspace is used solely for migration of x/params managed parameters
 	LegacySubspace exported.Subspace `optional:"true"`
+
+	EnableUnorderedTransactions bool `optional:"true"`
```

Comment:
How can we verify that this works properly?

Reply:
the legacy system test should fail since this is opt-in